README.md

```diff
@@ -78,7 +79,17 @@ npx @digitalbazaar/cborld-cli --help
 Examples:
 
 ```
-TBD
+# basic conversion of file, with verbose and diagnostic output
+./cborld encode --verbose --diagnose ../cborld/examples/note.jsonld -o note.cborld
+
+# decode file
+./cborld decode --verbose --diagnose ./note.cborld
+
+# output to hex string
+./cborld encode input.jsonld -o - | xxd -p -c 0 -
+
+# decode hex string
+xxd -r -p -c 0 input.hex | ./cborld decode -v -d -
```

Comment:
Local name is `cbor-cli.js`, but globally installed name is `cborld`, so the above doesn't work.

But this does!
```suggestion
cborld encode --verbose --diagnose ../cborld/examples/note.jsonld -o note.cborld

# decode file
cborld decode --verbose --diagnose ./note.cborld

# output to hex string
cborld encode input.jsonld -o - | xxd -p -c 0 -

# decode hex string
xxd -r -p -c 0 input.hex | cborld decode -v -d -
```

Reply:
Actually...use this one. It makes each command runnable in succession. We may also want to bring in the `note.jsonld` file into this repo to avoid downloading more stuff.

```suggestion
cborld encode --verbose --diagnose ../cborld/examples/note.jsonld -o note.cborld

# decode file
cborld decode --verbose --diagnose ./note.cborld
# ...will generate `out.jsonld`

# output to hex string
cborld encode out.jsonld -o - | xxd -p -c 0 - > out.hex

# decode hex string
xxd -r -p -c 0 out.hex | cborld decode -v -d -
# ...will generate `out.jsonld`
```